src/utils/awsConfig.js

```diff
@@ -0,0 +1,16 @@
+const getAccountId = (environment) => {
+  let accountID;
+  if (environment === 'development') {
+    accountID = '000000000000';
+  } else {
+    accountID = '242905224710';
+  }
+  return accountID;
+};
+
+const getAWSRegion = () => {
+  const region = process.env.AWS_REGION || process.env.DEFAULT_REGION || 'eu-west-1';
+  return region;
+};
+
+export { getAccountId, getAWSRegion };
```

Comment:
this won't work because of server/client side rendering -- we tried with Agi a few weeks ago. To get the environment, you will need to use an existing selector in redux for this.

Reply:
it works locally, but as soon as you deploy it in staging, you will see that the environment is set wrongly to development

Reply:
yeah thats what i do, I pass the environment from redux into this function?

Reply:
Yeah I'm just passing the variable to the function (which will be from redux in the places that call this function) so it should work?